### PR TITLE
add CONTRIBUTE.md file

### PR DESCRIPTION
**Pull request by yumalkhasyan**

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Contributing to this repository
```
Comment (zhidelev) [inline CONTRIBUTE.md:1]: Rename CONTRIBUTE.md to CONTRIBUTING.md

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Getting started
+
+Before you begin:
+-   This site is powered by Python 3.8.
+-   Have you read the [Best Practices for Managing Secrets](https://elearn.epam.com/courses/course-v1:EPAM+5SCSS+0620/courseware/4b94c749c309409ea878fb7916be316b/ae4553f7cd524229b42f260b0ac692ed/1?activate_block_id=block-v1%3AEPAM%2B5SCSS%2B0620%2Btype%40vertical%2Bblock%40c04f1498c7d04ac4bf87b652741d90bb)?
```
Comment (zhidelev) [inline CONTRIBUTE.md:7]: add "For EPAM employees only" before "Have you read the..."

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+-   Check out the [existing issues](https://github.com/jdi-testing/jdi-python/issues).
+
+### Don't see your issue? Open one
+
+Before you make your changes, check to see if an [issue exists](https://github.com/jdi-testing/jdi-python/issues) already for the change you want to make.
```
Comment (zhidelev) [inline CONTRIBUTE.md:12]: Rephrase to "Before you make your changes, check if an issue already exists for the change you want to make."

File: CONTRIBUTE.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+### Testing
+
+In order to test the project, run: 
+
+```bash
+pytest <path to the project>/tests --no-header --no-summary -q
+```
+
+Also don't forget to set up environment variable `TEST_PASSWORD`.
```
Comment (zhidelev) [inline CONTRIBUTE.md:26]: Put this phrase before ```bash...``` and rephrase to "Please don't forget to set up the environment variable TEST_PASSWORD."

File: CONTRIBUTE.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Contributing to this repository
+
+## Getting started
+
+Before you begin:
+-   This site is powered by Python 3.8.
```
Comment (zhidelev) [inline CONTRIBUTE.md:6]: Change to "Python 3.8 is used for this project."

File: CONTRIBUTE.md (hunk 4 above)
Comment (zhidelev) [reply CONTRIBUTE.md:26]: Also, add an example:
For Windows ```set TEST_PASSWORD=```

**Review by zhidelev — CHANGES_REQUESTED** [review]
Please don't pay any attention to failed Pypy build - it does not work with pywin32 for now 😿

**Comment by yumalkhasyan** [comment]
> Please don't pay any attention to failed Pypy build - it does not work with pywin32 for now 😿

Yep, I guessed so. I did my best yesterday trying to fix this but failed.
Although, do we really need those PyPy builds?

Also thanks for reviewing!

**Comment by zhidelev** [comment]
> 
> 
> > Please don't pay any attention to failed Pypy build - it does not work with pywin32 for now 😿
> 
> Yep, I guessed so. I did my best yesterday trying to fix this but failed.
> Although, do we really need those PyPy builds?
> 
> Also thanks for reviewing!

No problem 🙂 Ok, let's remove Pypy builds. We can add them back any moment in future 👍

**Review by zhidelev — COMMENTED** [review]
👍 
Remove Pypy builds and request Approval.

**Review by zhidelev — APPROVED** [review]
👍

**Pull request merged**